service/src/main/java/com/fit_goal/service/NotificatorService.java

```diff
@@ -0,0 +1,58 @@
+package com.fit_goal.service;
+
+import com.fit_goal.Audit;
+import com.fit_goal.api.Notificator;
+import com.fit_goal.domain.AuditDto;
+import com.fit_goal.domain.UserVerification;
+import com.fit_goal.enums.Notification;
+import com.fit_goal.util.MailSender;
+
+import javax.inject.Inject;
+import java.time.LocalDateTime;
+
+public class NotificatorService implements Notificator {
+
+    private final Audit audit;
+    private final MailSender mailSender;
+
+    @Inject
+    public NotificatorService(Audit audit, MailSender mailSender) {
+        this.audit = audit;
+        this.mailSender = mailSender;
+    }
+
+    @Override
+    public void register(UserVerification userVerification) {
+        //TODO create method for link generating
+        String link = String.format("user_service/verify/%s", userVerification.getVerificationLink());
```

Comment:
no need to use String.format, just concat the values

Reply:
refactored